src/js/button.js

```diff
@@ -12,6 +12,13 @@ import {assign} from './utils/obj';
  * @extends ClickableComponent
  */
 class Button extends ClickableComponent {
+  constructor(player, options) {
+    super(player, options);
+
+    this._mouseused = false;
```

Comment:
Is there any reason not stick this in `BigPlayButton` instead of the `Button` class?